bin/start-environment.sh

```diff
@@ -1,15 +1,8 @@
 #!/bin/sh
 
-if [[ ! -e 'integration-tests' ]]
-then
-  echo "Integration Tests not available, have you linked the directory? Check the README file for further details.\n"
-  exit 123
-fi
+source $(dirname $0)/internal/common.sh
+checkIntegrationTestsResourcesExists
 
-OPTIONS="-f integration-tests/docker-compose.yml -f docker-compose.yml --project-directory ."
+docker-compose up -d zap-proxy remote-webdriver citizen-frontend legal-frontend
 
-docker-compose ${OPTIONS} pull
```

Comment:
Pull just slowed down starting the environment too much for me so I decided to remove it. It is down to the user to pull latest images as it happens in integration-tests. It is now as simple as `docker-compose pull`.